/*#include "MyException.hpp"
#include <iostream>

MyException::MyException(const std::string& msg) noexcept(true)
: std::runtime_error(msg.c_str())
{
	std::cout << msg.c_str() << std::endl;
}*/
